src/dbnode/storage/bootstrap/bootstrapper/commitlog/source.go

```diff
@@ -234,6 +245,35 @@ func (s *commitLogSource) Read(
 				return bootstrap.NamespaceResults{}, err
 			}
 		}
+
+		// Read index snapshot files
+		indexSnapshotFiles, err := s.indexSnapshotFilesFn(
+			filePathPrefix,
+			ns.Metadata.ID(),
+		)
+		if err != nil {
+			return bootstrap.NamespaceResults{}, err
+		}
+		fsOpts := s.opts.CommitLogOptions().FilesystemOptions()
+		indexInfoFiles := s.readIndexInfoFilesFn(fsOpts.FilePathPrefix(), ns.Metadata.ID(),
```

Comment:
nit: think you already have variables for `fsOpts` and `filePathPrefix` created outside of this for loop